rocAL/rocAL/source/loaders/image/image_loader.cpp

```diff
@@ -250,7 +250,7 @@ ImageLoader::load_routine()
 
 bool ImageLoader::is_out_of_data()
 {
-    return (remaining_count() < _batch_size);
+    return (remaining_count() < 0);
```

Comment:
Why this change is added?

Reply:
The remainin image count was giving batch Size for the last batch. We need that to be 0 for last batch.Hence